facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -126,7 +134,7 @@ object FaciaPicker extends GuLogging {
       ("hasNoSlideshows", FrontChecks.hasNoSlideshows(faciaPage)),
       ("hasNoPaidCards", FrontChecks.hasNoPaidCards(faciaPage)),
       ("hasNoRegionalAusTargetedContainers", FrontChecks.hasNoRegionalAusTargetedContainers(faciaPage)),
-      ("hasNoSnapLinkCards", FrontChecks.hasNoSnapLinkCards(faciaPage)),
+      ("hasNoUnsupportedSnapLinkCards", FrontChecks.hasNoUnsupportedSnapLinkCards(faciaPage)),
```

Comment:
Often it's not great to update data point names so you can analyse it over time. This will drop off over 2 weeks, so not an issue at all, we'll just need to update the dashboard.

Reply:
> we'll just need to update the dashboard

yep, will do!

Reply:
have updated the dashboard now